mandatory/4-tax.js

```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
```

Comment:
Hi Mustafa I checked your code nice job all of tests has passsed. I like to use this method for getting decimal points that we need.